LexImpact : chiffres, réalisations et description du défi

DorineLam opened the pull request:
Première version

DorineLam committed:
Modif LexImpact

Première version

DorineLam committed:
Ajout image démo

DorineLam commented on _data/defis.yml:
```diff
@@ -239,22 +237,57 @@
+      description: "LexImpact POP s'appuyant sur des données protégées par le secret, il n'était pas possible d'ouvrir la version complète au grand public. Nous avons donc mis à disposition des citoyens le simulateur permettant de visualiser les effets sur des foyers fiscaux types."
+      url: https://www.leximpact.beta.gouv.fr
+      call_to_action: "Découvrir la version grand public"
+    - image: 
```
```suggestion
    - image:  /img/realisations/2019/LexImapct-Demos-5.jpg 
```

DorineLam committed:
Ajout des 3 autres images

DorineLam commented on _data/defis.yml:
```diff
@@ -239,22 +237,57 @@
+    - image: 
+      title: "Estimer précisément les impacts sur les déciles de population"
+      description: "Nous avons pu proposer en temps et heure du Projet de loi de finances 2020, une visualisation précise des impacts par décile de population, de l'impôt moyen payés par un foyer aux recettes précises par décile.
+    - image: 
```
```suggestion
    - image: /img/realisations/2019/LexImpact-version OPEN
```

DorineLam commented on _data/defis.yml:
```diff
@@ -239,22 +237,57 @@
+      description: "10 mois après : présentation du défi LexImpact."
+      show: false
+  projects:
+    - image: 
```
```suggestion
    - image: /img/realisations/2019/LexImpact-PhotoInterface.jpg
```

DorineLam commented on _data/defis.yml:
```diff
@@ -239,22 +237,57 @@
+    - image: 
+      title: "Le tableau de bord de LexImpact POP, accessibles aux parlementaires"
+      description: "Nous avons créé une interface facile d'utilisation permettant aux parlementaires d'estimer les impacts de leur amendement à l'Article 197 du CGI. En plus des impacts sur des cas types qu'ils peuvent configurer, ils ont une vision macro des effets sur la population et les recettes."
+    - image: 
```
```suggestion
    - image: /img/realisations/2019/LexImpact-Impacts macros.png
```

DorineLam commented on _data/defis.yml:
```diff
@@ -239,22 +237,57 @@
+      description: "Nous avons créé une interface facile d'utilisation permettant aux parlementaires d'estimer les impacts de leur amendement à l'Article 197 du CGI. En plus des impacts sur des cas types qu'ils peuvent configurer, ils ont une vision macro des effets sur la population et les recettes."
+    - image: 
+      title: "Estimer précisément les impacts sur les déciles de population"
+      description: "Nous avons pu proposer en temps et heure du Projet de loi de finances 2020, une visualisation précise des impacts par décile de population, de l'impôt moyen payés par un foyer aux recettes précises par décile.
```
```suggestion
      description: "Nous avons pu proposer en temps et heure du Projet de loi de finances 2020, une visualisation précise des impacts, de l'impôt moyen payé par un foyer par déciles, au nombre de "gagnants/perdants."
```

DorineLam commented on _data/defis.yml:
```diff
@@ -239,22 +237,57 @@
+    - amount: 4
+      description: "discours en séance publique parlent de LexImpact"
+  end_of_project_videos:
+    - id: x6y04yt
```
```suggestion
    - id: 
```

DorineLam commented on _data/defis.yml:
```diff
@@ -239,22 +237,57 @@
+      description: "discours en séance publique parlent de LexImpact"
+  end_of_project_videos:
+    - id: x6y04yt
+      platform: dailymotion
```
```suggestion
      platform: 
```

DorineLam reviewed (commented):
- Ajout des liens vers les images "projet"
- Suppression des infos vidéos du modèle antérieur
- correction description impacts macros

DorineLam committed:
Mise à jour description LexImpact

DorineLam committed:
Update _data/defis.yml

DorineLam committed:
Update _data/defis.yml

DorineLam committed:
Update _data/defis.yml

DorineLam committed:
Update _data/defis.yml

DorineLam committed:
Update _data/defis.yml

AntoineAugusti commented on _data/defis.yml:
```diff
@@ -239,22 +237,57 @@
+      description: "Nous avons créé une interface facile d'utilisation permettant aux parlementaires d'estimer les impacts de leur amendement à l'Article 197 du CGI. En plus des impacts sur des cas types qu'ils peuvent configurer, ils ont une vision macro des effets sur la population et les recettes."
+    - image: 
+      title: "Estimer précisément les impacts sur les déciles de population"
+      description: "Nous avons pu proposer en temps et heure du Projet de loi de finances 2020, une visualisation précise des impacts, de l'impôt moyen payé par un foyer par déciles, au nombre de "gagnants/perdants."
```
```suggestion
      description: "Nous avons pu proposer en temps et heure du Projet de loi de finances 2020, une visualisation précise des impacts, de l'impôt moyen payé par un foyer par déciles, au nombre de gagnants/perdants."
```

Tu ne peux pas avoir des guillemets doubles à l'intérieur du texte car sinon le YAML interprète ceci comme une fin d'un champ. Tu peux en revanche échapper un guillemet double `\"` si tu souhaites conserver des guillemets à l'intérieur.

milkosofi commented on _data/defis.yml:
```diff
@@ -239,22 +237,57 @@
+      description: "pourcent d'amendements concernés par l'outil ayant cité LexImpact dans l'exposé des motifs"
+    - amount: 84
+      description: "parlementaires et collaborateurs/collaboratrices ayant participé à une démonstration de l'outil"
+    - amount: 4
```
Quand vous mettez plus de 4 chiffres, l'affichage est très laid. Je vous suggère donc de choisir entre le 40% d'amendements qui citent LexImpact et les 4 discours. Je trouve que les 40% sont plus intéressants.

milkosofi committed:
Reformualtions des réalisations et chiffres

milkosofi commented on _data/defis.yml:
```diff
@@ -239,22 +237,61 @@
+    - image: 
+      title: "Estimer précisément les impacts sur les déciles de population"
+      description: "Au moment des discussions sur le Projet de loi finances 2020, nous avons pu proposer une visualisation précise des impacts, de l'impôt moyen payé par un foyer par déciles, au nombre de \"gagnants/perdants\"."
+      url: 
```
Vous pouvez mettre un lien vers les amendements qui citent votre outil.

milkosofi commented on _data/defis.yml:
```diff
@@ -239,22 +237,61 @@
+      title: "Impliquer, sensibiliser et former les parlementaires"
+      description: "Le produit livré par l'équipe a été accompagné par une présence régulière auprès des utilisateurs et par l'organisation de présentations pour former et sensibiliser les collaborateurs au chiffrage."
+      url: https://www.leximpact.beta.gouv.fr
+      call_to_action: "Découvrir la version grand public"
```
Vous remettez le lien vers le site leximpact.beta.gouv.fr, ce qui ne permet pas d'illustrer précisément la réalisation "formation". Est-ce que vous avez documenté votre travail de formation auprès de vos publics ? Avez-vous des supports de prise en main de l'outil ? Quelque chose de tangible et spécifique ?

milkosofi commented on _data/defis.yml:
```diff
@@ -239,22 +237,61 @@
+      url: 
+      call_to_action: 
+    - image: 
+      title: "Estimer précisément les impacts sur les déciles de population"
```
Ajouter "les impacts du PLF 2020" pour spécifier et différencier cette réalisation de la précédente.

milkosofi committed:
Modification de la description du défi

La version précédente disait deux fois la même chose, j'ai donc essayé de factoriser pour raccourcir et ne pas être redondant.

milkosofi commented:
Les images ne s'affichent pas : pourriez-vous vérifier ce qui ne va pas, Augustin et Dorine ?

milkosofi committed:
Suppression de gras et typos

milkosofi committed:
Modification de call to action 

Il faudra modifier les 3 autres call to action en fonction de ce vers quoi vous renverrez.

Commit:
Merge branch 'master' into patch-1

Commit:
Merge branch 'master' into patch-1

Commit:
Merge branch 'master' into patch-1

Commit:
Merge branch 'master' into patch-1

Commit:
Merge branch 'master' into patch-1

Commit:
Merge branch 'master' into patch-1

Commit:
Merge branch 'master' into patch-1

Commit:
Merge branch 'master' into patch-1

Commit:
Merge branch 'master' into patch-1

Commit:
Merge branch 'master' into patch-1

Commit:
Merge branch 'master' into patch-1

Commit:
Merge branch 'master' into patch-1

Commit:
Merge branch 'master' into patch-1

Commit:
Merge branch 'master' into patch-1

Commit:
Suppression du cinquième chiffre pour passer les tests

Commit:
Merge branch 'master' into patch-1

Commit:
Correction de noms de fichiers

Commit:
Correction des chemins d'accès d'images

Commit:
Correction d'espaces

Commit:
Ajout d'une image blank.jpg

bzg commented:
Attention : les noms d'images doivent être corrigés.
